FrontDesk/src/FrontDesk.Blazor/nginx.conf

```diff
@@ -0,0 +1,18 @@
+events { }
```

Comment:
I haven't used nginx like this before - can you explain what it's doing?

Reply:
Blazor WASM application needs host so I used Nginx to host it, better than use prerendered.